app/collectors/bucket.scala

```diff
@@ -7,72 +7,96 @@ import conf.AWS
 import controllers.routes
 import play.api.mvc.Call
 import software.amazon.awssdk.regions.Region
-import software.amazon.awssdk.services.s3.S3Client
 import software.amazon.awssdk.services.s3.model.{GetBucketLocationRequest, ListBucketsRequest, S3Exception, Bucket => AWSBucket}
+import software.amazon.awssdk.services.s3.{S3Client, S3Configuration}
 import utils.Logging
 
 import scala.jdk.CollectionConverters._
 import scala.language.{postfixOps, reflectiveCalls}
-import scala.util.Try
 import scala.util.control.NonFatal
 
 
-class BucketCollectorSet(accounts: Accounts) extends CollectorSet[Bucket](ResourceType("bucket"), accounts, Some(Regional)) {
+class BucketCollectorSet(accounts: Accounts) extends CollectorSet[Bucket](ResourceType("bucket"), accounts, Some(Global)) {
   val lookupCollector: PartialFunction[Origin, Collector[Bucket]] = {
     case amazon: AmazonOrigin => AWSBucketCollector(amazon, resource, amazon.crawlRate(resource.name))
   }
 }
 
 case class AWSBucketCollector(origin: AmazonOrigin, resource: ResourceType, crawlRate: CrawlRate) extends Collector[Bucket] with Logging {
 
+  // The `useArnRegionEnabled` flag enables us to receive data on buckets in all available AWS regions
+  // https://stackoverflow.com/questions/46769493/how-enable-force-global-bucket-access-in-aws-s3-sdk-java-2-0
+  val s3Configuration = S3Configuration.builder.useArnRegionEnabled(true).build
+
+  // The region of the S3 Client is hardcoded to EU-WEST-1, because AWS-Global does not work with S3.
+  // We decided to hardcode this here, instead of creating another enum for simplicity 
   val client = S3Client
     .builder
     .credentialsProvider(origin.credentials.provider)
-    .region(origin.awsRegionV2)
+    .region(Region.EU_WEST_1)
     .overrideConfiguration(AWS.clientConfig)
+    .serviceConfiguration(s3Configuration)
+    .build
+
+  // This second S3 client, with a region of US-EAST-1, gives us the correct createdTime value as documented here:
+  // https://stackoverflow.com/questions/54353373/getting-incorrect-creation-dates-using-aws-s3
+  val clientForCorrectCreatedTime = S3Client
```

Comment:
Could we combine `client` and `clientForCorrectCreatedTime`? That is, are we able to get all buckets using `Region.US_EAST_1`? A single client would make the code a little simpler.

Reply:
Thank you for this great question, Akash. It has made me reflect on the comments I have written and how I can make them clearer. I've made a change to the comment, which states why we cannot use one client. 

The reasoning is that we can only get the correct `createdTime` by using the US-EAST-1 region AND we can only receive data on all the buckets across all the AWS regions, if we use a region other than US-EAST-1 and AWS-GLOBAL. For this reason, I have used two clients.

Reply:
Worth saying that this approach is exactly how the AWS S3 web console works. It makes requests to different regions to obtain all of the data and then merges it all together.